tests/test_reader.py

```diff
@@ -3,53 +3,55 @@
 """
 
 import os
+import sys
+import logging
 import xml.etree.ElementTree as ET
 from glob import glob
 import pytest
 
+from pynxtools.dataconverter.convert import get_reader
 from pynxtools.dataconverter.helpers import (
     generate_template_from_nxdl,
-    validate_data_dict,
+    write_nexus_def_to_entry,
 )
+from pynxtools.dataconverter.validation import validate_dict_against
 from pynxtools.dataconverter.template import Template
 from pynxtools.definitions.dev_tools.utils.nxdl_utils import get_nexus_definitions_path
 
 from pynxtools_xps.reader import XPSReader
 
 
+READER = get_reader("xps")
+
+test_cases = [
+    ("phi_spe", "Phi .spe reader"),
+    ("phi_pro", "Phi .pro reader"),
+    ("specs_sle", "SPECS .sle reader"),
+    ("specs_xml", "SPECS .xml reader"),
+    ("specs_xy", "SPECS .xy reader"),
+    ("scienta_ibw", "Scienta .ibw reader"),
+    ("scienta_txt", "Scienta .txt export reader"),
+    ("vms_irregular", "Irregular VAMAS reader"),
+    ("vms_regular", "Regular VAMAS reader"),
+    # ("vms_txt_export", "Vamas txt export"),
```

Comment:
From you discord comment, I remember you have problem with vms file. Right?

Reply:
If the reader does not work with the vamas file, you can deactivate the option for running vamas example. if it is issue with test but generating nexus file with out any issue you can keep as it is.

Reply:
It does not work for this file, but I will have a fix coming in soon.